cfn.yaml

```diff
@@ -125,17 +125,46 @@ Resources:
           ConfigurationS3GetRole: !Sub arn:aws:iam::${MobileAccountId}:role/${CrossAccountBaseRoleName}-s3-${Stage}
           Stage: !Sub ${Stage}
       Description: lambda to replace the content-notifications-service
-      Handler: com.gu.mobile.content.notifications.Lambda::handler
+      Handler: com.gu.mobile.content.notifications.ContentLambda::handler
       MemorySize: 384
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java8
       Timeout: 60
 
-  EventSource:
+  ContentEventSource:
     Type: AWS::Lambda::EventSourceMapping
     Properties:
-      FunctionName: !Ref Lambda
+      FunctionName: !Ref ContentLambda
       Enabled: true
       EventSourceArn: !Sub arn:aws:kinesis:${AWS::Region}:${AWS::AccountId}:stream/${KinesisStreamName}-${Stage}
       StartingPosition: LATEST
 
+  LiveBlogLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub ${App}-liveblogs-${Stage}
+      Code:
+        S3Bucket:
+          Ref: DeployBucket
+        S3Key: !Sub ${Stack}/${Stage}/${App}/${App}.jar
+      Environment:
+        Variables:
+          ConfigurationBucket:
+            Ref: ConfigurationBucket
+          ConfigurationKey: !Sub ${Stage}/${App}/${App}.properties
+          ConfigurationS3GetRole: !Sub arn:aws:iam::${MobileAccountId}:role/${CrossAccountBaseRoleName}-s3-${Stage}
+          Stage: !Sub ${Stage}
+      Description: lambda to replace the content-notifications-service
```

Comment:
Lambda that sends push notifications when new key events are published on a liveblog